Services/Dashboard/Administration/classes/class.ilObjDashboardSettingsGUI.php

```diff
@@ -204,153 +317,107 @@ public function setSettingsSubTabs(string $a_active): void
-            $lng->txt("dash_presentation")
-        );
-
-        // sortation
-        $ops = $this->viewSettings->getAvailableSortOptionsByView($view);
-        $sortation_options = array_column(array_map(function ($k, $v) use ($lng) {
-            return [$v, $lng->txt("dash_sort_by_" . $v)];
-        }, array_keys($ops), $ops), 1, 0);
-        $avail_sort = $ui_factory->input()->field()->multiSelect($lng->txt("dash_avail_sortation"), $sortation_options)
-            ->withValue($this->viewSettings->getActiveSortingsByView($view));
-        $default_sort = $ui_factory->input()->field()->radio($lng->txt("dash_default_sortation"));
-        foreach ($sortation_options as $k => $text) {
-            $default_sort = $default_sort->withOption($k, $text);
-        }
-        $default_sort = $default_sort->withValue($this->viewSettings->getDefaultSortingByView($view));
-        $sec_sortation = $ui_factory->input()->field()->section(
-            $this->maybeDisable(["avail_sort" => $avail_sort, "default_sort" => $default_sort]),
-            $lng->txt("dash_sortation")
-        );
-
-        $form = $ui_factory->input()->container()->form()->standard(
-            $ctrl->getFormAction($this, $save_cmd),
-            ["presentation" => $sec_presentation, "sortation" => $sec_sortation]
-        );
-
-        return $form;
-    }
-
-
-    protected function saveViewCoursesGroups(): void
-    {
-        $this->saveViewSettings(
-            $this->viewSettings->getMembershipsView(),
-            "editViewCoursesGroups"
+            $title
         );
     }
 
-    protected function editViewFavourites(): void
+    protected function savePresentation(): void
     {
-        $main_tpl = $this->tpl;
-        $tabs = $this->tabs_gui;
-        $ui_renderer = $this->ui_renderer;
-
-        $tabs->activateTab("settings");
-        $this->setSettingsSubTabs("view_favourites");
+        $request = $this->request;
+        $lng = $this->lng;
+        $ctrl = $this->ctrl;
 
-        $view = $this->viewSettings->getSelectedItemsView();
+        $form = $this->getViewForm(self::VIEW_MODE_PRESENTATION);
+        $form = $form->withRequest($request);
+        $form_data = $form->getData();
 
-        $form = $this->getViewSettingsForm($view);
+        if (!$this->canWrite()) {
+            $this->tpl->setOnScreenMessage('failure', $this->lng->txt('no_permission'), true);
+            $this->editPresentation();
+        }
 
-        $main_tpl->setContent($ui_renderer->render($form));
-    }
 
-    protected function saveViewFavourites(): void
-    {
-        $this->saveViewSettings(
-            $this->viewSettings->getSelectedItemsView(),
-            "editViewFavourites"
-        );
+        foreach ($form_data as $view => $view_data) {
+            $this->viewSettings->storeViewPresentation(
+                $view,
+                $view_data['default_pres'],
+                $view_data['avail_pres'] ?: []
```

Comment:
Do you want to proof `== false` or `=== null` here?
If second, use `??` since its more strict.